src/ui/logic.js

```diff
@@ -1,6 +1,69 @@
 "use strict";
-let stageID = 1;
-const stageLogic = function (stage) {
-  if (stage === 1) {
+import { userObject } from "./form.js";
+//let stageID = 1;
+const userObjectParsed = JSON.parse(localStorage.user);
```

Comment:
Here is the answer to your question https://stackoverflow.com/questions/66190250/why-does-the-whole-javascript-file-execute-even-though-im-only-exporting-a-funct. Javascript first runs through the modules to create/declare the functions for them later to be imported e.g. This means you have to put this inside the function to avoid execution. Also remember to use `getItem` to get item from localStorage https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage